libvalhalla.pc.in

```diff
@@ -1,10 +1,10 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=${exec_prefix}/@libdir@
-includedir=${prefix}/@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: libvalhalla
 Description: valhalla c++ library
-Version: @VERSION@
-Libs: -L${libdir} -lvalhalla @deplibs@
+Version: @PROJECT_VERSION@
```

Comment:
is this actually defined somewhere? the info is located in `valhalla/valhalla.h.in`

Reply:
Good question, no it wasn't. It was an assumption and a wrong one at that, fixed it. I assumed Valhalla was setting the project version in CMake, but it seems it gets it from CPack and has different variables.